### DISCUSSION TITLE
Lesson 11: NFT not showing on Opensea

### DISCUSSION THREAD

**Discussion by andreitoma8** [smartcontractkit/full-blockchain-solidity-course-py#715] [Q&A] [answered]
I created the first Smart Contract, deployed it and created a NFT. Contract is showing on etherscan( https://rinkeby.etherscan.io/address/0xda3b2e73f0bc3c2e2e6ab41a707471a1850ef6a3 ) and I can even see the NFT in my wallet, but OpenSea gives me the 404 error when I acces the link and I cannot see any NFTs in Account page. I lould love to know if this is something from my end or a problem with OpenSea.

**Comment by andreitoma8** [comment, marked as answer]
SOLVED: It seems to be a OpenSea problem as I was able to view my NFT on Metamask IOS App after manually importing the NFT.

**Reply by cromewar** [reply]
I'm glad you solved so quickly.

**Reply by LintonAchmad** [reply]
Having the same issue - how do I import the NFT to metamask on browser?

**Reply by andreitoma8** [reply]
Sorry for the late answer, but for people who might be interested in the future: 
1. Find the address of you deployed Smart Contract.
2. Don't forget to select the appropriate chain in your MetaMask App! 
3. In MetaMask Mobile, tap on the 'NFTs' tab, scroll down and tap on the "+ ADD NFTs" link. Paste the NFT's address from your clipboard into the "Address" box.
4. Paste the ID of the NFT into the box marked "ID" in MetaMask Mobile, tap on the "ADD" button, and your NFTs should appear under the NFTs tab.

**Reply by alexkalen** [reply]
I did this but I cant really see an image
